app/views/layouts/application.html.erb

```diff
@@ -40,7 +37,7 @@
       href="<%= @comments_rss_link[:href] %>" />
   <% end %>
 </head>
-<body>
+<body data-current-user-id="<%= @user.id %>">
```

Comment:
Unfortunately, the build is failing - if there isn't a user logged in, `@user` is `nil`. This would leave it to be the empty string, so the js would `parseInt` to `NaN`, which is falsey for the `Lobsters.curUser` checks... seem OK to you?

```suggestion
<body data-current-user-id="<%= @user.try(:id) %>">
```

Reply:
looks good. I take care of it next week

Reply:
Cool, thanks!